addon/styles/addon.scss

```diff
@@ -44,6 +44,12 @@ $frost-modal-overflow-shadow-color: rgba(0, 0, 0, .13);
   }
 }
 
+.frost-modal-form {
```

Comment:
Still don't understand why we're creating a top level selector - `.frost-modal-dialog` does not have one.  If this is actually desired then we should make our implementation consistent.

Reply:
fixed